server/internal/server/models.go

```diff
@@ -1231,6 +1266,19 @@ func isModelLoaded(m *store.Model) bool {
 	return toLoadingStatus(m.LoadingStatus) == v1.ModelLoadingStatus_MODEL_LOADING_STATUS_SUCCEEDED
 }
 
+func activationCategory(isBase bool, status v1.ActivationStatus) int {
```

Comment:
not sure `activationCategory` is the right word here. `modelSortingCategory` is better?